cg/meta/workflow/microsalt.py

```diff
@@ -155,7 +155,7 @@ def get_parameters(self, sample_obj: models.Sample) -> Dict[str, str]:
         method_sequencing = self.lims_api.get_sequencing_method(sample_id)
         if method_sequencing:
             method_sequencing, _ = method_sequencing.split(" ", 1)
-        priority = "research" if sample_obj.priority == 0 else "standard"
+        priority = "research" if sample_obj.priority_int == 0 else "standard"
```

Comment:
It would be great if "research" and "standard" in this line also were named constant